src/components/HomePage/Banner.js

```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles(theme => ({
 	container: {
 		backgroundColor: theme.palette.primary.dark,
 		height: 'auto',
-		padding: 25,
+		padding: 0,
```

Comment:
```suggestion
```
Padding by default on divs (which this style is applied to) is already 0, so we can just remove this line.